src/contexts/WalletContext/WalletProvider.tsx

```diff
@@ -17,6 +17,12 @@ const WalletProvider: React.FunctionComponent<WalletProviderProps> = ({
   const [name, setName] = useState<WalletName | null>(null);
   const [balance, setBalance] = useState<Record<string, BigNumber>>({});
   const [required, setRequired] = useState<boolean>(false);
+  const [sessionId, setSessionId] = useState<string>("");
+
+  useEffect(() => {
+    setSessionId((Math.random()).toString(20).substring(2, 10));
```

Comment:
lets use the package uuid from npm. yarn add -E uuid@9.0.0. https://www.npmjs.com/package/uuid 

then simple use their utility to generate the random id.

Reply:
@ioanaroceanu2000 
Why was this resolved? 
Can you share the commit where this was addressed? 
If you have it locally push it please!

Best practice is to resolve the comment after you pushed the fix. The share it via message in the conversation. 

Example message: "@reviewer fixed in <commit_hash>"

Reply:
resolved 9bb0958572bdde864d23200e7f12fd6c19910057